### PR TITLE
Correct comment listing statements that can be EXPLAINed

### PR DESCRIPTION
**Pull request by jseldess**
The comment was missing `CREATE`, `DROP`, and `ALTER TABLE`, all of which can be explained.

---

This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/7676)

### CODE REVIEW

File: sql/explain.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ const (
 var explainStrings = []string{"", "debug", "plan", "trace", "types"}
 
 // Explain executes the explain statement, providing debugging and analysis
-// info about a DELETE, INSERT, SELECT or UPDATE statement.
+// info about an ALTER TABLE, CREATE, DELETE, DROP, INSERT, SELECT, or UPDATE statement.
```
Comment (tamird) [inline sql/explain.go]: this list may change - is it really worth keeping? perhaps "info about the wrapped statement." is better.

**Comment by knz** [comment]
I concur with @tamird. Let's remove the specific items entirely from the comment.

**Comment by jseldess** [comment]
OK, removed mention of specific statements. 

---

Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks pending.

---

*[sql/explain.go, line 44 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/7676#-KM51U_vVYcJZFhIhiwM-r1-44:-KM57nMUUvnvmO7zlBfM:-499146009) ([raw file](https://github.com/cockroachdb/cockroach/blob/a239334048d4420d18b6f06f1749a681016f4895/sql/explain.go#L44)):*
<details><summary><i>Previously, tamird (Tamir Duberstein) wrote…</i></summary>
> this list may change - is it really worth keeping? perhaps "info about the wrapped statement." is better.
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/7676#-:-KM57oQPNXKqmeOHOHHq:157649734)*

**Comment by knz** [comment]
LGTM

**Pull request merged**